_posts/2011-11-28-les-cast-codeurs-podcast-episode-49-devoxx-2011-les-dix-ans.md

```diff
@@ -46,11 +46,11 @@ Devoxx France du 18 au 20 avril 2012 à Paris [http://www.devoxx.fr/](http://www
 [http://www.touilleur-express.fr/2011/11/16/lhistoire-de-devoxx-france-2012/](http://www.touilleur-express.fr/2011/11/16/lhistoire-de-devoxx-france-2012/)  
 Startup weekend [http://startupweekend.org/](http://startupweekend.org/)
 
-## Enregistrement live du 50eme épisode au ParisJUG
```

Comment:
tu es sûr que 50ème n'est pas valable (vs 50e) ? Moi petit j'ai apris ème je suis quasi certain.

Reply:
Je m'étais aussi posé la question car j'avais le même souvenir. Mais comme la correction venait de l'excellent spellchecker de Intellij IDEA, [Grazie](https://blog.jetbrains.com/idea/2020/07/new-spellchecker-implementation-in-intellij-idea-2020-2/), j'ai fait quelques recherches et suis tombé sur http://www.academie-francaise.fr/abreviations-des-adjectifs-numeraux. Si c'est l'académie français qui le dit c'est que ça doit être vrai ;). Après théoriquement le 'e' devrait être surélevé.

Si tu ne connais pas Grazie et que tu utilises Intellij IDEA je te conseille d'y jeter un coup d’œil. Je ne suis pas très bon en grammaire / orthographe et ce plugin m'a vraiment changé la vie. Je crois qu'il est maintenant packagé par défaut dans Intellij IDEA / WebStorm... et qu'il n'y a plus qu'à ajouter le français dans `File | Settings | Editor | Proofreading`.